script/Common/00_00.txt

```diff
@@ -172,7 +172,7 @@ Panicking, the girl darts out of the room to call for a doctor.
 [sha:0147d9ffc03795f6a14bede26557467864df16d4]{
 -- Page 11, Offset 139.
 -- 急いでいるのはわかるけど、病院で走ってはいけません。
-I get that she's in a rush, but you're not supposed to run in hospitals.
+I get that she's in a rush, but you aren't supposed to run in hospitals.
```

Comment:
Original feels better to me, also ngrams
https://books.google.com/ngrams/graph?content=you%27re+not+supposed%2C+you+aren%27t+supposed&year_start=1800&year_end=2019&corpus=26&smoothing=3&direct_url=t1%3B%2Cyou%20%27re%20not%20supposed%3B%2Cc0%3B.t1%3B%2Cyou%20are%20not%20supposed%3B%2Cc0#t1%3B%2Cyou%20're%20not%20supposed%3B%2Cc0%3B.t1%3B%2Cyou%20are%20not%20supposed%3B%2Cc0